data/crac-io/crac-io-json/src/main/java/com/powsybl/openrao/data/craciojson/JsonSerializationConstants.java

```diff
@@ -280,55 +282,58 @@ public static Unit deserializeUnit(String stringValue) {
         }
     }
 
-    public static String serializeSide(Side side) {
-        switch (side) {
-            case LEFT:
-                return LEFT_SIDE;
-            case RIGHT:
-                return RIGHT_SIDE;
-            default:
-                throw new OpenRaoException(String.format("Unsupported side %s", side));
-        }
+    public static String serializeSide(TwoSides side) {
```

Comment:
Maybe you can simply use TwoSides.toString and TwoSides.valueOf to simplify this and deserializeSide ?

Reply:
Ok I will do that, I just note that this will change the case: left/right -> ONE/TWO

Reply:
plus, why not use integers 1 and 2 instead of strings, to make the CRAC lighter

Reply:
done